src/main/resources/db/migration/V1_16__update_service_ticket_code_for_sscs.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM judicial_service_code_mapping WHERE service_id ='21' OR service_id ='55';
```

Comment:
you can move these scripts to the above created file, so that we can avoid having an extra file